documentation/content/en/chapters/upgrade_guide.xml

```diff
@@ -386,23 +386,24 @@
 			</orderedlist>
 		</section>
 
-		<section id="update_automatic">
+		<section id="update_advanced">
 			<sectioninfo>
 				<authorgroup>
 					<author>
 						<othername>Noxwizard</othername>
 					</author>
 				</authorgroup>
 			</sectioninfo>
-			<title>Automatic Update</title>
-			<para>This update method is only recommended for installations with modifications to core phpBB files. This package detects changed files automatically and merges in changes if needed.</para>
-			<para>A number of automatic update files are available, and you should choose the one that corresponds to the version of the board that you are currently running. For example, if your current version is 3.3.0 and you are updating to 3.3.1, you need the phpBB-3.3.0_to_3.3.1.zip/tar.bz2 file.</para>
+			<title>Advanced Update (Expert users)</title>
+			<para>This update method should only be used for installations with modifications to core phpBB files. If you simply use Extensions or custom Styles and have not modified core files, please use the Full Package update.</para>
```

Comment:
Maybe we should add something that this used to be the automatic updater and that we strongly advice against changing core files. (And I'd like to state this update option will be removed in the future)

Reply:
Done